src/main/java/orm/DefaultEntityPersister.java

```diff
@@ -17,8 +21,32 @@ public DefaultEntityPersister(QueryBuilder queryBuilder, QueryRunner queryRunner
 
     @Override
     public <T> T persist(T entity) {
-        return queryBuilder.insertIntoValues(entity, queryRunner)
+        var tableClassifier = new RelationFields<>(entity);
+
+        T rootEntity = queryBuilder.insertIntoValues(entity, queryRunner)
                 .returnAsEntity();
+
+        // 연관관계 필드 저장
+        List<RelationField> valuedRelationList = tableClassifier.getValuedRelationList();
+        for (RelationField relationField : valuedRelationList) {
+            persistRelations(relationField);
+        }
+        return rootEntity;
+    }
+
+    // 연관관계 필드 저장
+    private <T> void persistRelations(RelationField relationField) {
```

Comment:
```suggestion
    private void persistRelations(RelationField relationField) {
```
여기서는 제네릭이 이제 쓰이지 않네요 :)